app/app.go

```diff
@@ -168,6 +204,7 @@ var (
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
 		claimmoduletypes.ModuleName:    {authtypes.Minter, authtypes.Burner, authtypes.Staking},
 		allocmoduletypes.ModuleName:    {authtypes.Minter, authtypes.Burner, authtypes.Staking},
+		wasm.ModuleName:                {authtypes.Burner},
```

Comment:
I'm guessing this is only for native coins that are sent to the wasmd module account?

Reply:
I believe a contract can only burn tokens sent to them